packages/core/src/dialects/abstract/query.d.ts

```diff
@@ -129,54 +129,44 @@ export class AbstractQuery {
   /**
    * Returns the unique constraint error message for the associated field.
    *
-   * @param field {string} the field name associated with the unique constraint.
+   * @param field the field name associated with the unique constraint.
    *
-   * @returns {string} The unique constraint error message.
+   * @returns The unique constraint error message.
    * @private
    */
   private getUniqueConstraintErrorMessage(field: string): string;
 
   /**
    * Checks if the query type is RAW
-   *
-   * @returns {boolean}
    */
   isRawQuery(): boolean;
 
   /**
    * Checks if the query type is VERSION
-   *
-   * @returns {boolean}
    */
   isVersionQuery(): boolean;
 
   /**
    * Checks if the query type is UPSERT
-   *
-   * @returns {boolean}
    */
   isUpsertQuery(): boolean;
 
   /**
    * Checks if the query type is INSERT
-   *
-   * @returns {boolean}
    */
   isInsertQuery(results?: unknown[], metaData?: unknown): boolean;
 
   /**
    * Sets auto increment field values (if applicable).
    *
-   * @param results {Array}
-   * @param metaData {object}
-   * @returns {boolean}
+   * @param results
+   * @param metaData
+   * @returns
```

Comment:
Do we need these? They do not contain more information than the type below so wouldn't they be inferred from that?